initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -88,24 +135,30 @@ elif [ -e /etc/redhat-release -o -e /etc/SuSE-release ]; then
 fi
 
 echo "Removing Directory - ${path}"
-rm -rf ${path}
+removedir ${path}
+
+if [ -n "${mklink}" -a -n "${lpath}" ] ; then
```

Comment:
The original dir is left so perms are maintained.  The re-created soft-link perms should follow the original dir perms.